cmd/build_test.go

```diff
@@ -24,6 +25,12 @@ func TestBuildCmd(t *testing.T) {
 	})
 
 	t.Run("Success build cmd with --artifacts-dir", func(t *testing.T) {
+		defFunc := osMkdirAll
+		osMkdirAll = os.MkdirAll
+		defer func() {
+			osMkdirAll = defFunc
```

Comment:
If we use default value of `os.MkdirAll`. Maybe we don't need this defer things?

Reply:
https://github.com/screwdriver-cd/sd-local/pull/15/files#diff-c20737afa75dbf45a65e7088758c44e7R41
By default, we mock `osMkdirAll`.
we want to use `os.MkdirAll` only in this case

Reply:
I'm sorry i don't understand. Do we need mock of osMkdirAll. I feel we can just use `os.MkdirAll`.

Reply:
I understood.